fortanix-sgx-tools/src/bin/ftxsgx-runner.rs

```diff
@@ -81,7 +81,7 @@ fn main() -> Result<(), Error> {
         None => (),
     }
 
-    enclave_builder.arg(file);
+    enclave_builder.arg("enclave");
```

Comment:
When you launch a process through `std::process::Command`, you don't get to specify arg 0. Should be the same here, this should be handled in `enclave-runner`.